### PR TITLE
Initial Mozilla WoT Gateway interface

### PR DESCRIPTION
**Pull request by hobinjk-ptc**
- [ ] Guide user through local token process
- [ ] Properly list discovered things

Probably not:
- [ ] Discover gateway.local if available (might not since it requires disabling certificate verification)

### CODE REVIEW

File: interfaces/mozillaWot/MozillaWotInterface.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
+const fetch = require('node-fetch');
+const WebSocket = require('ws');
+
+const server = require('../../../../libraries/hardwareInterfaces');
```
Comment (benptc) [inline interfaces/mozillaWot/MozillaWotInterface.js:4]: Some other interfaces (e.g. kepware) use the syntax `require('@libraries/hardwareInterfaces');` - not sure if one way is better than the other but pointing it out

File: interfaces/mozillaWot/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+layout: doc
+title: Philips Hue Interface
+permalink: /docs/vuforia-spatial-basic-interfaces-addon/interfaces/philipsHue
+---
+
+## Philips Hue Interface
```
Comment (benptc) [inline interfaces/mozillaWot/README.md:7]: Are you planning to update this from Philips Hue to Mozilla WoT?

File: interfaces/mozillaWot/config.html [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+    <div style="display:inline-block"></div> <!-- graphic design is my passion -->
+    <div class="humanName button green three">thing</div>
+</template>
+
+<body>
+
+<div id="introduction">
+    If your gateway URL is missing please use this to manually configure your
+    interface with the necessary values. To add a token, visit your gateway's
+    Developer Settings page to issue a new local authorization.
```
Comment (benptc) [inline interfaces/mozillaWot/config.html:236]: Nice instructions 🙂

File: interfaces/mozillaWot/MozillaWotInterface.js (hunk 1 above)
Comment (hobinjk-ptc) [reply interfaces/mozillaWot/MozillaWotInterface.js:4]: This one is the eventual standard since electron doesn't play nice with `@libraries`

**Pull request merged**